### PR TITLE
Installs rspec, configures shoulda matchers

### PR DESCRIPTION
**Pull request by raptorf1**
# [PT Story](https://www.pivotaltracker.com/story/show/165794204)

* Bundles rspec gem to the project
* Installs rspec to the project
* Configures shoulda matchers
* Removes comments for spec files

### CODE REVIEW

**Comment by faraznaeem** [comment]
Please add PT link and a better title for this PR

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  # arbitrary gems may also be filtered via:
+  # config.filter_gems_from_backtrace("gem name")
+end
+
+Shoulda::Matchers.configure do |config|
+  config.integrate do |with|
+    with.test_framework :rspec
+    with.library :rails 
+  end
+end
```
Comment (faraznaeem) [inline spec/rails_helper.rb:31]: Please clean out comments

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  #     --seed 1234
+  config.order = :random
+
+  # Seed global randomization in this process using the `--seed` CLI option.
+  # Setting this allows you to use `--seed` to deterministically reproduce
+  # test failures related to randomization by passing the same `--seed` value
+  # as the one that triggered the failure.
+  Kernel.srand config.seed
+=end
+end
```
Comment (faraznaeem) [inline spec/spec_helper.rb:17]: Please clean out the comments

**Review by faraznaeem — CHANGES_REQUESTED** [review]

File: spec/rails_helper.rb (hunk 1 above)
Comment (raptorf1) [reply spec/rails_helper.rb:31]: done

File: spec/spec_helper.rb (hunk 2 above)
Comment (raptorf1) [reply spec/spec_helper.rb:17]: done

**Comment by raptorf1** [comment]
# [PT Story](https://www.pivotaltracker.com/story/show/165794204)

Clear out comments from `rails_helper.rb` and `spec_helper.rb`

**Review by faraznaeem — APPROVED** [review]

**Pull request merged**